client/js/components/launchdarkly.js

```diff
@@ -0,0 +1,27 @@
+import LDClient from 'ldclient-js';
+import uuidV1 from 'uuid/v1';
+
+const LDUserKey = 'LDUserKey';
+
+let uuid = window.localStorage.getItem(LDUserKey);
+
+if (!uuid) {
+  uuid = uuidV1();
+  window.localStorage.setItem(LDUserKey, uuid);
+}
+
+const user = {
+  key: uuid,
+  anonymous: true
+};
+
+const ldclient = LDClient.initialize('58e63fe26b7519095c7484cb', user);
+
+const getFlags = new Promise((resolve, reject) => {
```

Comment:
Was just chatting to @jamesgorrie and realised we don't get `Promise`s out of the box with our babel configuration. We can add them from core-js in the same way as we do with `Object.assign` (at the top of [app.js](https://github.com/wellcometrust/wellcomecollection.org/blob/master/client/js/app.js#L2))